src/assets/auth.ts

```diff
@@ -12,38 +12,37 @@
  * Removal or modification of this copyright notice is prohibited.
  */
 import { getLisk32AddressFromAddress } from '@liskhq/lisk-cryptography';
-
-import { MODULE_NAME_AUTH } from '../constants';
-import { genesisAuthStoreSchema } from '../schemas';
 import {
 	Account,
 	AuthAccountEntry,
 	AuthStoreEntry,
 	AuthStoreEntryBuffer,
 	GenesisAssetEntry,
 } from '../types';
+import { MODULE_NAME_AUTH } from '../constants';
+import { genesisAuthStoreSchema } from '../schemas';
 
 const keyMapper = (key: Buffer) => key.toString('hex');
 const keyComparator = (a: Buffer, b: Buffer) => a.compare(b);
 
-export const addAuthModuleEntry = async (accounts: Account[]): Promise<GenesisAssetEntry> => {
-	const authSubstoreEntries: AuthStoreEntryBuffer[] = await Promise.all(
-		accounts.map(async (account: Account) => {
-			const authObj: AuthAccountEntry = {
-				numberOfSignatures: account.keys.numberOfSignatures,
-				mandatoryKeys: account.keys.mandatoryKeys.sort(keyComparator).map(keyMapper),
-				optionalKeys: account.keys.optionalKeys.sort(keyComparator).map(keyMapper),
-				nonce: String(account.sequence.nonce),
-			};
+export const getAuthModuleEntry = async (account: Account): Promise<AuthStoreEntryBuffer> => {
+	const authObj: AuthAccountEntry = {
+		numberOfSignatures: account.keys.numberOfSignatures,
+		mandatoryKeys: account.keys.mandatoryKeys.sort(keyComparator).map(keyMapper),
+		optionalKeys: account.keys.optionalKeys.sort(keyComparator).map(keyMapper),
+		nonce: String(account.sequence.nonce),
+	};
 
-			return {
-				storeKey: account.address,
-				storeValue: authObj,
-			};
-		}),
-	);
+	return {
+		storeKey: account.address,
+		storeValue: authObj,
+	};
+};
 
-	const sortedAuthSubstoreEntries: AuthStoreEntry[] = authSubstoreEntries
+export const addAuthModuleEntry = async (
```

Comment:
Not an error, just a small inconsistency. I think you sort all the other entries in `CreateAsset` and then pass already sorted data to `add...ModuleEntry`, only for Auth module you pass unsorted data and sort in `addAuthModuleEntry`. 
Maybe it makes sense to move the sorting logic into `createAsset.ts`.